app/src/main/java/com/nmakademija/nmaakademija/fragment/ScheduleFragment.java

```diff
@@ -84,9 +84,25 @@ private void setScheduleItems(List<ScheduleEvent> scheduleEvents) {
         }
 
         Collections.sort(scheduleItems, new ScheduleItemComparator());
+        int i = 0;
+        Date now = new Date();
+        while (i < scheduleItems.size()) {
+            if (scheduleItems.get(i).getDate().before(now))
+                i = i + 1;
+            else
+                break;
+        }
```

Comment:
Something like this could save 5 lines of code.

int i = 0;
Date now = new Date();
for (i = 0; i < scheduleItems.size () && !scheduleItems.get(i).getDate().before(now); ++i)